.rubocop.yml

```diff
@@ -7,22 +7,11 @@ require:
 
 # our custom config
 # TODO: Uncomment it after fix rubocop-todo RSpec/MultipleExpectations
-RSpec/MultipleExpectations:
-  # Max: 1
-  Exclude:
-    - 'spec/features/**/*'
-
 RSpec/DescribeClass:
   Exclude:
-    - spec/features/**/*
     - spec/requests/**/*
     - spec/system/**/*
 
-RSpec/ExampleLength:
```

Comment:
I don't think setting `RSpec/ExampleLength` Max to default (5) makes sense now. We'll get a lot of violations if we do this and it's not optimal to fix them now.

Reply:
Oh I removed that by accident. Thanks for pointing that out.